Source/missiles.cpp

```diff
@@ -2533,22 +2541,11 @@ void AddFireman(int mi, int sx, int sy, int dx, int dy, int midir, int8_t mienem
 
 	anim = &monster[id].MType->Anims[MA_WALK];

```

Comment:
fixed, but i hesitated cause it's not directly related to this pr. But I did it anyway 😉

Reply:
Mainly mentioned this variable as it was only used in the function you created to pull out shared code. With the change to use an animation ID it's been completely removed anyway so that's great 😀